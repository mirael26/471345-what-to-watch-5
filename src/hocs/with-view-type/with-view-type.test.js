import React from "react";
import renderer from "react-test-renderer";

import withViewType from "./with-view-type";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withViewType(MockComponent);

it(`Should withViewType render correctly`, () => {
  const noop = () => {};
  const tree = renderer
    .create(
        <MockComponentWrapped
          viewType={`overview`}
          onTabClick={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
