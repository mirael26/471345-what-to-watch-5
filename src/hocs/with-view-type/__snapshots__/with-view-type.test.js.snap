// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should withViewType render correctly 1`] = `<div />`;
